query/src/main/scala/filodb/query/exec/ScalarBinaryOperationExec.scala

```diff
@@ -58,12 +63,21 @@ case class ScalarBinaryOperationExec(queryContext: QueryContext,
                        querySession: QuerySession)
                       (implicit sched: Scheduler): Task[QueryResponse] = {
     val rangeVectors : Seq[RangeVector] = Seq(ScalarFixedDouble(params, evaluate))
-    Task.eval { // not async
-      QueryResult(
-        queryContext.queryId, resultSchema, rangeVectors, QueryStats(), QueryWarnings(),
-        querySession.resultCouldBePartial,
-        querySession.partialResultsReason
-      )
+    Kamon.runWithSpan(Kamon.currentSpan(), false) {
```

Comment:
Why override `execute` here at all? All scalars do this, I guess, so `ScalarRangeVectors` can be returned?

Reply:
Yes